aea/cli/core.py

```diff
@@ -284,9 +301,13 @@ def generate_wealth(ctx: Context, type_):
     )
     wallet = Wallet(private_key_paths)
     try:
+        balance = _get_balance(ctx.agent_config, wallet, type_)
```

Comment:
can be removed here as balance is only needed in `_wait_funds_release` and so put this line there.